src/pages/eth2/get-involved.js

```diff
@@ -0,0 +1,721 @@
+              Here's all the ways you can help with Ethereum and future
+              Eth2-related efforts.
+            </Subtitle>
+          </HeroContainer>
+        </HeroCard>
+      </Content>
+      <Content>
+        <H2 id="#clients">Run beacon chain clients</H2>
+        <p>
+          Key to Ethereum's long term security is a strong distribution of
+          clients. A client is software that runs the blockchain, checking
+          transactions and the creation of new blocks. Each client has its own
+          features, so choose one based on what you're comfortable with.
+        </p>
+        <StyledCardContainer>
+          {clients.map((client, idx) => {
+            return (
+              <ProductCard
+                key={idx}
+                url={client.url}
+                background={client.background}
+                image={client.image}
+                name={client.name}
+                description={client.description}
+              />
+            )
+          })}
+        </StyledCardContainer>
+      </Content>
+      <StyledGrayContainer>
+        <Content>
+          <Row>
+            <LeftColumn>
+              <H2 id="#bug-bounty">Go bug hunting</H2>
+              <p>
+                Find and report bugs in Eth2 upgrade specifications or the
+                clients themselves. You can earn up to $50,000 USD and earn a
+                place on the leaderboard.
+              </p>
+              <p>A bug might be:</p>
+              <p>
+                <ul>
+                  <li>specification non-compliance issues</li>
+                  <li>finality breaking bugs</li>
+                  <li>denial of service (DOS) vectors</li>
+                  <li>and more...</li>
+                </ul>
+              </p>
+              <ButtonLink to="/en/eth2/bug-bounty/">Go bug hunting</ButtonLink>
+            </LeftColumn>
+            <LeaderboardContainer>
+              <Leaderboard content={bugHunters} />
+            </LeaderboardContainer>
+          </Row>
+        </Content>
+      </StyledGrayContainer>
+      <Staking>
+        <StyledCalloutBanner
+          image={data.rhino.childImageSharp.fluid}
+          title="Stake your ETH"
+          description="You can now stake your ETH to help secure the beacon chain. "
+        >
+          <div>
+            <ButtonLink to="/en/eth2/staking/">Stake ETH</ButtonLink>
+          </div>
+        </StyledCalloutBanner>
+      </Staking>
+    </Page>
+  )
+}
+
+export default GetInvolvedPage
+
+export const Clients = graphql`
+  fragment Clients on File {
+    childImageSharp {
+      fixed(width: 80) {
+        ...GatsbyImageSharpFixed
+      }
+    }
+  }
+`
+
+export const Hunters = graphql`
+  fragment Hunters on File {
+    childImageSharp {
+      fixed(width: 40) {
+        ...GatsbyImageSharpFixed
+      }
+    }
+  }
+`
+
+export const query = graphql`
+  query {
+    rhino: file(relativePath: { eq: "eth2/eth2_rhino.png" }) {
```

Comment:
@ryancreatescopy with the risk of getting too pedantic, let's keep the kebab-case convention for images (& all filenames), i.e. eth2-rino.png